### PR TITLE
docs: update cli deprecations

### PR DESCRIPTION
**Pull request by alan-agius4**
Add new CLI deprecations and remove old ones

### CODE REVIEW

**Review by kyliau — APPROVED** [review]

File: aio/content/guide/deprecations.md [hunk 1]
```diff
@@ -437,32 +437,12 @@ This section contains a complete list all of the currently deprecated CLI flags.
...
-| `setResolveHook`                | <!--v8--> v10     | Not used within projects. Used with Tooling API only. Not Yarn PnP compatible and not used in the Angular CLI. Use Node.js [require.resolve](https://nodejs.org/api/modules.html#modules_require_resolve_request_options).|
-| `ResolveOptions`                | <!--v8--> v10     | Not used within projects. Used with Tooling API only. Not Yarn PnP compatible and not used in the Angular CLI. Use Node.js [require.resolve](https://nodejs.org/api/modules.html#modules_require_resolve_request_options).|
-| `terminal`                      | <!--v8--> v10     | Unused implementation of terminal codes (color).                                |
-| `isObservable`                  | <!--v8--> v10     | Not used within projects. Used with Tooling API only. Use `isObservable` function from the `rxjs` package.|
+| `extractCss`                    | <!--v11--> v13     | No longer required to disable CSS extraction during development.               |
+| `i18nFormat`                    | <!--v9--> TBD     | Format is now automatically detected.                                           |
```
Comment (jelbourn) [inline aio/content/guide/deprecations.md:441]: I'm not a huge fan of saying TBD here. Why not just say `v12` since they're not being cut in v11?

Comment (alan-agius4) [reply aio/content/guide/deprecations.md:441]: Because this depends on when VE will be removed from the framework.

Comment (alan-agius4) [reply aio/content/guide/deprecations.md]: @jelbourn, pushed the changes.

**Review by jelbourn — APPROVED** [review]
LGTM

**Review by jelbourn — APPROVED** [review]
LGTM

Reviewed-for: docs-packaging-and-releasing

**Pull request closed without merging**